public/scripts/upload.js

```diff
@@ -365,6 +365,13 @@ function updateFileListItem(itemNode, imageUpload) {
   }
 }
 
+function handleColorPickersTooltip() {
+  tippy("[data-tippy-content]", {
+    placement: "bottom-end",
+    theme: "light-border",
+  });
+}
+
```

Comment:
May I know how the original tooltips works?

And if we are using bootstrap, did we explore the bootstrap tooltips instead of loading in tippy.js.

Reply:
> how the original tooltips works?

IIUC, there were no tooltip. I added it for better UX

> if we are using bootstrap

I haven't used that in my styling, so went for the first google result which demo & API LGTM

> did we explore the bootstrap tooltips instead of loading in tippy.js

Agree with this, maybe should've checked bootstrap first. Do you think we should remove tippy?

Reply:
okok, lemme check check, I prefer removing bootstrap if nothing using it

> prefer to use bootstrap tooltips

May I know why? Is it just because the project was using it?

Reply:
just checked, `ListItem.astro` is using. Will remove tippy & use bootstrap

Reply:
> May I know why? Is it just because the project was using it?

Yes and we can't randomly add lib/way of doing things to make the project become *legacy* project.